packages/flutter/lib/src/utils/parse_live_grid.dart

```diff
@@ -151,6 +151,12 @@ class _ParseLiveGridWidgetState<T extends sdk.ParseObject>
       ),
     );
     return GridView.builder(
+        reverse: widget.reverse,
+        padding: widget.padding,
+        physics: widget.scrollPhysics,
+        controller: widget.scrollController,
+        scrollDirection: widget.scrollDirection,
+        // shrinkWrap: widget.shrinkWrap,
```

Comment:
Why is this line committed?

Reply:
> Why is this line committed?

It shouldn't was just testing something.